cloud-sql/mysql/servlet/README.md

```diff
@@ -70,10 +70,10 @@ mvn clean package appengine:deploy
 See the [Cloud Run documentation](https://cloud.google.com/run/docs/configuring/connect-cloudsql)
 for more details on connecting a Cloud Run service to Cloud SQL.
 
-1. Build the container image:
+1. Build the container image using [Jib](https://cloud.google.com/java/getting-started/jib):
 
   ```sh
-  gcloud builds submit --tag gcr.io/[YOUR_PROJECT_ID]/run-mysql
+    mvn compile com.google.cloud.tools:jib-maven-pluginbuild \ -Dimage=gcr.io/[YOUR_PROJECT_ID]/run-mysql
   ```
 
 2. Deploy the service to Cloud Run:
```

Comment:
Can you add flags:
```
    --platform managed \
    --allow-unauthenticated \
   --region [REGION]
```

For step 3, we prefer using `--update-env-vars` and if you want more clarity you can use the flag multiple times like:
```
--update-env-vars CLOUD_SQL_CONNECTION_NAME=[INSTANCE_CONNECTION_NAME] \
--update-env-vars DB_USER=[MY_DB_USER],DB_PASS=[MY_DB_PASS],DB_NAME=[MY_DB]
```

Reply:
Done + combined steps 2+3.